lib/src/products_page.dart

```diff
@@ -41,7 +47,39 @@ class _Products extends StatelessWidget {
 class _Product extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
-    return Text('Product Item');
+    return Card(
+      child: Column(
+        children: <Widget>[
+          AspectRatio(
+            aspectRatio: 20.0 / 11.0,
+            child: Image.asset('assets/images/T-shirt1.jpeg'),
+          ),
+          Container(
+            width: 200,
```

Comment:
Why specify width here? It should fill available space?

Reply:
been remove with commit 26ccfb62d95c765ae078e7a068c78eeeb9c23457